src/mip/bound_strengthening.jl

```diff
@@ -0,0 +1,91 @@
+@doc raw"""
+
+The behavior of bound_strengthening!() depends on the ordering of the variables.
```

Comment:
Move this to a note at the bottom of the docstring:

```suggestion
Note: The behavior of `bound_strengthening!` depends on the ordering of the variables.
```